client/src/entrypoints/admin/telepath/widgets.js

```diff
@@ -170,18 +171,122 @@ class DraftailRichTextArea {
     this.options = options;
   }
 
-  render(container, name, id, initialState, parentCapabilities) {
-    const originalOptions = this.options;
-    const options = { ...originalOptions };
-    const capabilities = parentCapabilities || new Map();
+  getCapabilityOptions(originalOptions, parentCapabilities) {
+    // Given original options object, return the options overrides
+    // that account for its contextual abilities (splitting or adding additional blocks)
+    const options = {};
+    const capabilities = parentCapabilities;
     const split = capabilities.get('split');
+    const addSibling = capabilities.get('addSibling');
+    let blockCommands = [];
     if (split) {
-      options.controls = options.controls ? [...options.controls] : [];
-      options.controls.push(
-        // eslint-disable-next-line no-undef
-        draftail.getSplitControl(split.fn, !!split.enabled),
-      );
+      options.controls = originalOptions.controls
+        ? [...originalOptions.controls]
+        : [];
+      options.controls.push({
+        meta: window.draftail.getSplitControl(split.fn, !!split.enabled),
+      });
+
+      const blockGroups =
+        addSibling && addSibling.enabled && split.enabled
+          ? addSibling.blockGroups
+          : [];
+      // Create commands for splitting + inserting a block. This requires both the split
+      // and addSibling capabilities to be available and enabled
+      blockCommands = blockGroups.map(([group, blocks]) => {
+        const blockControls = blocks.map((blockDef) => {
+          const blockMax = addSibling.getBlockMax(blockDef.name);
+          return {
+            icon: `#icon-${blockDef.meta.icon}`,
+            description: blockDef.meta.label,
+            type: blockDef.name,
+            render: ({ option }) => {
+              // If the specific block has a limit, render the current number/max alongside the description
+              const limitText =
+                typeof blockMax === 'number'
+                  ? ` (${addSibling.getBlockCount(blockDef.name)}/${blockMax})`
+                  : '';
+              return `${option.description}${limitText}`;
+            },
+            onSelect: ({ editorState }) => {
+              const result = window.draftail.splitState(editorState);
```

Comment:
Most likely missed as part of rebase – we need to re-export this for it to work.